FEATURES.md

```diff
@@ -0,0 +1,17 @@
+# Proposing Features
+
+A feature request or proposal is an issue opened on a repo in opensearch-project that states an end-user problem, and proposes a solution in the form of a feature. It should meet the following criteria.
+
+- Is a feature.
+- Fits in the scope of the repo and project.
+- Is sufficiently distinct from existing features.
+- Doesn’t overlap or conflict with ongoing work.
+- Is sufficiently atomic.
+
+The primary goal of a feature request is to brainstorm the solutions with the community early. The feature proposal should be opened in the idea stage, or as soon as a problem has been encountered. A technical or an implementation design is not required, but it's recommended to include ideas of possible approaches if you have any. Feature requests are usually labelled as `enhancement` or `feature`, depending on the repo, and often use the [recommended template](.github/ISSUE_TEMPLATE/FEATURE_REQUEST_TEMPLATE.md).
+
+Repo maintainers regularly [triage](TRIAGING.md) new feature requests to determine whether they are valid. There should be no expectation that anyone will pickup the feature request and work on it, including the person who opened it.
+
+Most features can be elevated to the project [roadmap](https://github.com/orgs/opensearch-project/projects/1) by tagging them with a "roadmap" label and a target version number. See [maintaining the project roadmap](RESPONSIBILITIES.md#manage-roadmap) for more information.
+
+Finally, features should be given time before appreciable work begins, allowing for community to voice opinions.
```

Comment:
Do we want to make this a little stronger, putting a "review period of no less than a week" for features?  (adding in a date field in the template?)

Reply:
I feel pretty strongly not wanting to dictate timelines for teams.